source/transfer_list.rst

```diff
@@ -657,5 +658,58 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+
+.. _tpm_evlog_entry:
+
+TPM info table entry layout (XFERLIST_TPM_INFO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This entry type holds TPM info of the platform.
+
+.. _tab_tpm_info:
+.. list-table:: TPM info type layout
+   :widths: 2 2 2 8
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **5**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
+     - 0x4
+     - 0x4
+     - sizeof (base_address) + sizeof (evlog_size) + sizeof (crb_base_address) + sizeof (crb_size)
```

Comment:
Since the size of this entry must be constant I would recommend writing it like the tag_id description (i.e. "The data_size field must be set to **24**.")